OpenUtau.Core/Classic/VoicebankLoader.cs

```diff
@@ -386,6 +388,13 @@ public class VoicebankLoader {
                 oto.Error = $"{trace}\nFailed to parse overlap. Format is {format}.";
                 return oto;
             }
+            if (!IsTest) {
+                string path = Path.Combine(Path.GetDirectoryName(trace.file), oto.Wav);
+                if (!File.Exists(path)) {
```

Comment:
It's wasteful to check for every oto line. It would be better to parse all lines first, then check files.

Reply:
I updated it.